composer.json

```diff
@@ -2,6 +2,7 @@
     "name": "nekland/woketo",
     "license": "MIT",
     "type": "library",
+    "description": "I let you fill this part ;) ",
```

Comment:
Let's make some SEO on packagist <3. I guess we can add `A WebSocket library for PHP.` here.